sections/main-product.liquid

```diff
@@ -131,24 +141,6 @@
             {%- endif -%}
           {%- when 'custom_liquid' -%}
             {{ block.settings.custom_liquid }}
-          {%- when 'collapsible_tab' -%}
```

Comment:
You went a bit too far here 😅, my incomplete explanation is at fault I think. Sorry!
I meant that you should remove the _instances_ of the collapsible_tab that are related to "Actual item pictured", not the entire block definition. Basically this can be done in either the admin UI after deployment, or you can find the usages of collapsible tab that are related to "Actual item pictured" (try searching this through the entire project to find them for example) in the .json templates under the `dawn/templates/` directory, I can see 6 templates that use it.

The other collapsible tabs should remain working. See image below :)


<img width="1267" alt="image" src="https://user-images.githubusercontent.com/10142342/199181610-41d63c8d-8d77-4aef-b024-d9d6783250a8.png">

For completeness' sake, to remove the entire block definition for collapsible_tab, there are actually more settings for it around line 864 in this file, and in the locale files as well. Try searching for _collapsible_tab_ through the entire project too.

Reply:
Aaaa, will do this! Thanks!

Reply:
Should now be fixed.